source/guide/3.2/reference-diamond-snmp-integration.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: bt_wiki
+title: Diamond SNMP integration
+category: Reference
+publish: true
+abstract: "Reference on how to use diamond monitoring with SNMP device"
+pageord: 1020
+---
+
+{%summary%}
+An extension to the diamond plugin that adds support for monitoring SNMP metrics on remote machines.
+The examples usage and necessary types are in [cloudify-diamond-snmp-extension](https://github.com/cloudify-cosmo/cloudify-diamond-snmp-extension)
+{%endsummary%}
+
+{%note title=Note%}
+Here you can find [Diamond plugin guide](plugin-diamond.html)
```

Comment:
Here you can find [Diamond plugin guide](plugin-diamond.html) -->
See [Diamond Plugin](plugin-diamond.html) for general Diamond plugin usage.